src/query/models/types.go

```diff
@@ -99,6 +100,14 @@ type TagOptions interface {
 	IDSchemeType() IDSchemeType
 }
 
+type FetchOverrideOptions struct {
+	ShouldOverride bool
```

Comment:
Hehe, I like the avoiding the heap allocation by using a bool instead of making it a pointer type (for optionality).

Reply:
I really wish Go just had bloody optionals (i.e. unwrapping) baked in to the language in a zero alloc way.